src/twister2/report/yaml_test_reporting_plugin.py

```diff
@@ -11,6 +11,9 @@
 @pytest.hookimpl(tryfirst=True)
 def pytest_report_teststatus(report, config):
 
+    if not hasattr(config, 'twister_config'):
```

Comment:
Why `twister_config` is checked here? Shouldn't be `config.getoption('twister') or config.getini('twister')` as in other plugins (just to be consistent)?

Reply:
`pytest_report_teststatus` is called after `pytest_configure`, where twister flag is checked and `twister_config` is added to pytest config, so it does not matter which flag we are checking. Checking `twister_config` is easier (I do that also in `plugin.py: pytest_collection_modifyitems` ), but if you think, that it could be better to check twister flag from cmd (or ini file) to be more consistent with rest of code (checking in `pytest_configure`), I can change it.